db/migrate/20170504145330_sync_whitehall_public_updated_at.rb

```diff
@@ -0,0 +1,29 @@
+class SyncWhitehallPublicUpdatedAt < ActiveRecord::Migration[5.0]
+  def change
+    schema_names = %w(gone redirect special_route speech statistical_data_set take_part topical_event_about_page)
```

Comment:
I'm wondering what advantage limiting by `schema_name` gives us, what if there is a mismatched time in an edition that is not part of these schemas?